### PR TITLE
reword add-to-app landing page and add video

### PR DESCRIPTION
**Pull request by xster**

### CODE REVIEW

File: src/docs/development/add-to-app/index.md [hunk 1]
```diff
@@ -14,38 +14,61 @@ render a part of your app's UI in Flutter. Or just to run shared Dart logic.
...
 ## Supported features
 
 ### Add to Android applications
 
 - Auto-build and import the Flutter module by adding a Flutter SDK hook to
   your Gradle script; or
-- Build your Flutter module into a generic [Android AAR](https://developer.android.com/studio/projects/android-library)
-- Android Studio Android/Flutter co-editing and module creation/import wizard
-- Java and Kotlin host apps supported
+- Build your Flutter module into a generic [Android AAR](https://developer.android.com/studio/projects/android-library).
```
Comment (blasten) [inline src/docs/development/add-to-app/index.md]: nit: Android archive (AAR)

File: src/docs/development/add-to-app/index.md [hunk 2]
```diff
@@ -14,38 +14,61 @@ render a part of your app's UI in Flutter. Or just to run shared Dart logic.
...
+- Android Studio Android/Flutter co-editing and module creation/import wizard.
+- Java and Kotlin host apps supported.
+- Flutter modules can use [Flutter plugins](https://pub.dev/flutter) to interact
+  with the platform. Android plugins should be [migrated to the V2 plugins APIs](/docs/development/packages-and-plugins/plugin-api-migration)
+  for best add-to-app correctness.
+- Support for Flutter debugging and Stateful Hot Reload by using `flutter attach`
```
Comment (blasten) [inline src/docs/development/add-to-app/index.md]: q: does this need to be capitalized?

File: src/docs/development/add-to-app/index.md [hunk 3]
```diff
@@ -14,38 +14,61 @@ render a part of your app's UI in Flutter. Or just to run shared Dart logic.
...
+  from IDEs or the command line to connect to an app containing Flutter
```
Comment (blasten) [inline src/docs/development/add-to-app/index.md]: nit: period

File: src/docs/development/add-to-app/index.md [hunk 4]
```diff
@@ -14,38 +14,61 @@ render a part of your app's UI in Flutter. Or just to run shared Dart logic.
...
-### Android and iOS
+- Build your Flutter module into a generic [iOS Framework](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFrameworks/Concepts/WhatAreFrameworks.html).
+- Objective-C and Swift host apps supported.
 - Flutter modules can use [Flutter plugins](https://pub.dev/flutter) to interact
-  with the platform
-- Flutter modules support Stateful Hot Reload by using `flutter attach` from
-  IDEs or the command line to connect to an app containing Flutter
+  with the platform.
+- Support for Flutter debugging and Stateful Hot Reload by using `flutter attach`
+  from IDEs or the command line to connect to an app containing Flutter
```
Comment (blasten) [inline src/docs/development/add-to-app/index.md]: ditto: missing period

**Comment by xster** [comment]
Done, thanks.

**Pull request merged**